apps/test/unit/makerlab/TouchSensorTest.js

```diff
@@ -0,0 +1,105 @@
+/** @file Test the TouchSensor maker component which wraps Touchpad */
+import {expect} from '../../util/configuredChai';
+import sinon from 'sinon';
+import {EventEmitter} from 'events'; // provided by webpack's node-libs-browser
+import TouchSensor from '@cdo/apps/makerlab/TouchSensor';
+
+const DOWN = 'down';
+const HOLD = 'hold';
+const UP = 'up';
+
+describe('TouchSensor', function () {
+  let fakeTouchPad, touchSensor, spy;
```

Comment:
Looks like you're right and this is called out by [AirBnB rule 13.2](https://github.com/airbnb/javascript#variables--one-const), enforced via [ESLint one-var](http://eslint.org/docs/rules/one-var.html).

We've never enforced this - ESLint finds 8017 violations of this rule in `/apps` right now, and unfortunately it's not autofixable.

Personally, I prefer the `"one-var": ["error", {"initialized": "never"}],` variant on this rule that enforces one-var-per-declaration for initialized variables, but permits lists of uninitialized variables like this one.  For what it's worth, there are only 62 violations of _that_ rule right now.  It might be a good place to start if we want to begin enforcing this.